.circleci/config.yml

```diff
@@ -9,21 +9,26 @@ jobs:
       MATPLOTLIBRC: tests
     steps:
       - checkout
+      - run:
+          name: Update to latest `nox`
+          command: |
+            pip uninstall --yes nox-automation
+            pip install "nox >= 2018.9.14"
       - run:
           name: Check that all Cython generated files have been updated
-          command: nox -s "update_generated(check=True)"
+          command: nox -s "update_generated-3.7(check=True)"
```

Comment:
I'm not a big fan of this enforced name change by `nox` and discussion continues at https://github.com/theacodes/nox/issues/118